src/main/java/com/beerjournal/datamodel/repository/mongo/CollectableObjectsMapper.java

```diff
@@ -0,0 +1,67 @@
+package com.beerjournal.datamodel.repository.mongo;
+
+import com.beerjournal.datamodel.entity.BottleEntity;
+import com.beerjournal.datamodel.entity.CanEntity;
+import com.beerjournal.datamodel.entity.CapEntity;
+import com.beerjournal.datamodel.entity.CollectableObjectEntity;
+import com.beerjournal.datamodel.entity.LabelEntity;
+import com.beerjournal.datamodel.model.Bottle;
+import com.beerjournal.datamodel.model.Can;
+import com.beerjournal.datamodel.model.Cap;
+import com.beerjournal.datamodel.model.CollectableObject;
+import com.beerjournal.datamodel.model.Label;
+
+public class CollectableObjectsMapper {
```

Comment:
Z początku nie chciałem jakkolwiem wiązać obiektów modelowych z entity, ale faktycznie jak się spojrzy na ną klasę to boli xD także jak najbardziej dobry pomysł, podobny zabieg zrobię również z updatem (można się trochę doczepić że logika jest przerzucana na pojo, ale chwilowo update samych obiektów jest banalny więc to również przeniosę)